lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -2,26 +2,24 @@
     <div
         [attr.data-automation-id]="'card-select-label-' + property.key"
         class="adf-property-label"
+        [ngClass]="{'adf-property-value-editable' : isEditable()}"
     >{{ property.label | translate }}</div>
     <div class="adf-property-field">
         <div
             *ngIf="!isEditable()"
-            class="adf-select-item-padding adf-property-value"
+            class="adf-property-value adf-property-read-only"
             [attr.data-automation-id]="'select-readonly-value-' + property.key"
             data-automation-class="read-only-value">{{ (property.displayValue | async) | translate }}
         </div>
         <div *ngIf="isEditable()">
-            <mat-form-field class="adf-select-item-padding-editable adf-property-value">
+            <mat-form-field class="adf-property-value" [ngClass]="{'adf-property-value-editable' : isEditable()}">
```

Comment:
same here, we should avoid calling functions in html

Reply:
Done